src/tests/ftest/io/nvme_pool_capacity.py

```diff
@@ -115,14 +120,20 @@ def ior_thread(self, pool, oclass, api, test, flags, results):
             results.put("FAIL")
 
     def test_create_delete(self, num_pool=2, num_cont=5, total_count=100,
-                           scm_size=400000000000, nvme_size=700000000000):
+                           scm_size=100000000000, nvme_size=300000000000):
         """
         Test Description:
-            This method is called with
-            num_pool parameter to run following test case
-            scenario's.
-            Use Cases
-             1. Create Pool/Container and destroy them. Check Space.
+            This method is used to create/delete pools
+            for a long run. It verifies the NVME free space
+            during this process.
+            Args:
+                num_pool: Total pools for running test
```

Comment:
We usually put type next to parameter. e.g.,
`num_pool (int): Total pools for running test`

Reply:
Sure. Will add it...